### DISCUSSION TITLE
useDataRefresh sometimes returns 502

### DISCUSSION THREAD

**Discussion by germangetgloby** [sergiodxa/remix-utils#140] [Q&A] [answered]
Hello! 

We are using `useDataRefresh` to make a request every second, and it usually works well. However, we are getting a 502 error in about 1 out of every 20 requests. The request itself is quick and lightweight. Do you have any suggestions for how to resolve this issue?

![image (1)](https://user-images.githubusercontent.com/102976995/210246003-a8fcc985-9b61-4bdc-83ed-f0518a839800.jpg)

**Comment by sergiodxa** [comment, marked as answer]
A 502 Bad Gateway is probably your server not supporting the amount of continuous requests, I’m not sure how you could fix it aside of scaling up your server.

Note that useDataRefresh will be deprecated soon once v1.10 of Remix is released with useRevalidator support.
